x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -143,26 +145,57 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
     </>
   ) : (
     <>
-      <ExpandableEventTitle
-        isAlert={isAlert}
-        loading={loading}
-        ruleName={ruleName}
-        handleOnEventClosed={handleOnEventClosed}
-      />
-      <EuiSpacer size="m" />
-      <ExpandableEvent
-        browserFields={browserFields}
-        detailsData={detailsData}
-        event={expandedEvent}
-        isAlert={isAlert}
-        isDraggable={isDraggable}
-        loading={loading}
-        rawEventData={rawEventData}
-        timelineId={timelineId}
-        timelineTabType={tabType}
-        hostRisk={hostRisk}
-        handleOnEventClosed={handleOnEventClosed}
-      />
+      {isIsolateActionSuccessBannerVisible && (
```

Comment:
Ohhhhh wow. I would not have guessed I needed to add this here. I actually had "force" the display of the prior code block by adding `isFlyoutView || isHostIsolationPanelOpen` on line 102, which clearly would not have been the right approach.